crates/biome_js_analyze/tests/specs/nursery/useCollapsedIf/valid.js

```diff
@@ -0,0 +1,18 @@
+if (condition && anotherCondition) {
```

Comment:
ditto. Please check the test file.
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/test/no-lonely-if.mjs